war/src/main/js/pluginSetupWizardGui.js

```diff
@@ -212,6 +212,15 @@ var createPluginSetupWizard = function(appendTarget) {
 						'<button type="button" class="close" aria-label="Close"><span aria-hidden="true">&times;</span></button>');
 				}
 
+				// add Jenkins version
+				if(translations.installWizard_jenkinsVersionTitle) { // wait until translations available
```

Comment:
Does this really make sense? Isn't *Jenkins 2.72* pretty much locale independent?

Also, this means that for languages without a translation it's completely missing, right? Not sure that's a great idea for something like this.